modules/javafx.graphics/src/main/native-glass/mac/a11y/AccessibleBase.m

```diff
@@ -62,7 +62,7 @@ + (void) initializeRolesMap {
     [rolesMap setObject:@"JFXMenuItemAccessibility" forKey:@"MENU"];
     [rolesMap setObject:@"JFXMenuItemAccessibility" forKey:@"MENU_ITEM"];
     [rolesMap setObject:@"JFXMenuItemAccessibility" forKey:@"RADIO_MENU_ITEM"];
-    [rolesMap setObject:@"JFXMenuItemAccessibility" forKey:@"CHECK_MENU_ITEM"];
```

Comment:
Is the removal of CHECK_MENU_ITEM intentional ?

Reply:
No, that was an accident.

Reply:
... and I totally missed that!
Question: how come none of the tests failed?  Do we have test coverage in this area?

Reply:
I do not think so - and i do not think the non-manual test would be possible and even with the manual test it will pass because removal of binding just means that old code will be used and the old code still works - until Apple pulls the plug on old API.